lib/analysis_options.flutter.yaml

```diff
@@ -0,0 +1,20 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
The base is different between dart and flutter. But with the analysis options, we can't include multiple definitions...